src/test/java/org/openrewrite/java/migrate/javax/AddCommonAnnotationsDependenciesTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.javax;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.config.Environment;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.java.Assertions.javaVersion;
+import static org.openrewrite.java.Assertions.mavenProject;
+import static org.openrewrite.java.Assertions.srcMainJava;
+import static org.openrewrite.java.Assertions.version;
+import static org.openrewrite.maven.Assertions.pomXml;
+
+@SuppressWarnings("LanguageMismatch")
```

Comment:
Surprised about this one; is there any way we can shuffle the language hints below to make it not necessary to suppress warnings?
```suggestion
```

Reply:
Again it's a copy paste ! 
I'll remove it.

From [AddJaxbDependenciesTest](https://github.com/openrewrite/rewrite-migrate-java/blob/v2.11.0/src/test/java/org/openrewrite/java/migrate/javax/AddJaxbDependenciesTest.java)